jenkins-jobs/scripts/copy-images.sh

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
```

Comment:
Could the job (https://github.com/debezium/debezium/blob/master/jenkins-jobs/sync-images.yaml) be updated to use this script and extended to provide all-tags copy like this script https://github.com/debezium/docker-images/blob/master/image-sync/sync.sh ?

Reply:
I'm not quite sure I follow what you mean

Reply:
Well, we have a job that syncs Docker Hub images to Quay.io - effectivelly doing more or less the same your script does. It is more adnnced in ability to copy all tags but otherwise much more limited.

So I was think about using your script to do the same task.